cmd/juju/application/deploy_test.go

```diff
@@ -435,7 +436,7 @@ func (s *DeploySuite) TestDeployFromPath(c *gc.C) {
 func (s *DeploySuite) TestDeployFromPathUnsupportedSeries(c *gc.C) {
 	path := testcharms.RepoWithSeries("bionic").ClonedDirPath(c.MkDir(), "multi-series")
 	err := s.runDeploy(c, path, "--series", "quantal")
-	c.Assert(err, gc.ErrorMatches, `series "quantal" not supported by charm, supported series are: focal,jammy,bionic`)
+	c.Assert(err, gc.ErrorMatches, `multi-series is not available on the following series: quantal not supported`)
```

Comment:
The existing code just prints the error instead of a slice of series, https://github.com/juju/juju/blob/4520638e6cc76701fc0362771bcdb37aa512d48c/cmd/juju/application/deployer/deployer.go#L635 I can change it.